notebooks/community/pipelines/google_cloud_pipeline_components_cloud_natural_language_pipeline.ipynb

```diff
@@ -0,0 +1,826 @@
+{
```

Comment:
<p>why is CLASS_NAMES empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Users are expected to fill in the class names of their dataset here. Unfortunately there is no Colab form for string lists.</p>